### ISSUE TITLE
Uncaught ReferenceError: Cannot access 'me' before initialization

### ISSUE THREAD

**Issue by smiffy6969** [uiwjs/react-markdown-preview#243]
Hi,

I am using this package on a VITE + React 18.2.0 setup (TS) and things seem to work fine on local.

Upon compile and push to static web app on azure, I get the following issue...

 @uiw-64952b32.js:26 Uncaught ReferenceError: Cannot access 'me' before initialization
    at @uiw-64952b32.js:26:10652

Which resolves to...

`const ra = me.forwardRef((e,t)=>{
    var {prefixCls: n="wmde-markdown wmde-markdown-color", className: r, source: i, style: u, disableCopy: l=!1, skipHtml: a=!0, onScroll: s, onMouseOver: o, pluginsFilter: c, rehypeRewrite: m, wrapperElement: d={}, warpperElement: p={}} = e
      , S = Dn(e, Ol)
      , b = me.useRef(null);
    Zt.useImperativeHandle(t, ()=>be({}, e, {
        mdp: b
    }), [b, e]);`

Any thouhgts on what this could be, working fine on a vite dev local, but vite build and push is coming up with this.

thanks

Paul

**Comment by jaywcjlove** [comment]
@smiffy6969 Please provide a reproducible example for debugging purposes.

**Comment by smiffy6969** [comment]
No problem at all, thanks for the fast reply...

https://github.com/smiffy6969/markdown-test

npm run dev

will run @ http://localhost:5173/

markdown fine

npm run build
npm run preview

will run @ http://localhost:4173/

@uiw-64952b32.js:26 Uncaught ReferenceError: Cannot access 'me' before initialization

**Comment by jaywcjlove** [comment]
![image](https://github.com/uiwjs/react-markdown-preview/assets/1680273/3f3fe167-9417-4052-9aef-1dae72eb6386)

![image](https://github.com/uiwjs/react-markdown-preview/assets/1680273/dcd885d2-31b1-4755-bdff-4ba9856ada80)

@smiffy6969 I haven't reproduced your error.

**Comment by smiffy6969** [comment]
how interesting!

**Comment by smiffy6969** [comment]
So you are correct, it is fine, my cache from my actual project was holding on to the error.... so i still have the error on my project, so maybe i can try and reproduce why... i am beginning to wonder if its due to a lazy loaded component

**Comment by smiffy6969** [comment]
well, that was bizarre to say th eleast, thanks so much for your effort, spent an hour or two going through this one at a time....

I ended up deleting my lock file and hey presto, a reinstall of node modules fixed it.

Best i can guess is a conflict with react-markdown, I had removed that (was trying that first) but lockfile still had in. Doesn't sound like it should be an issue, but i did have a commented out import of it in a file. Best answer i can give i think.

Thanks for your time and thanks for a great project.

paul
